guides/guides.json

```diff
@@ -192,6 +192,12 @@
         "icon": "vsf",
         "as": "/guides/vsf",
         "href": "/guides/[...guide]"
+      },
+      {
+        "text": "Vue Storefront Instance",
```

Comment:
It's weird that we have one guide called "Vue Storefront" and another called "Vue Storefront Instance". Maybe they should be "Vue Storefront 1" and "Vue Storefront 2+"?

Reply:
@markbrocato I'm not sure about the naming here yet. I think the one that we have in docs is reflecting VSF Nuxt projects bootstrapped from vsf cli. The demo I was asked to work over is reflecting the vsf app created with "installer" way described [here](https://docs.vuestorefront.io/guide/installation/linux-mac.html#user-friendly-installation) and that task was called "VSF Instance". So probably it's worth to call it being something like _"Vue Storefront Demo (from installer)"_ or _"... (from source)"_.